src/stapi_fastapi/models/opportunity.py

```diff
@@ -22,6 +22,8 @@ class OpportunityRequest(BaseModel):
     # TODO: validate the CQL2 filter?
     filter: CQL2Filter | None = None
     model_config = ConfigDict(strict=True)
```

Comment:
move `model_config` to the end and have one blank line between it and the fields definitions